Makefile

```diff
@@ -19,11 +30,8 @@ default:
 	@echo "  redis-store-cli  Opens a Redis CLI to the store Redis server"
 	@echo "  clear-caches     Clear Redis caches"
 	@echo "  shell            Opens a Bash shell"
-	@echo "  currentshell     Opens a Bash shell into existing running 'web' container"
 	@echo "  test             Runs the Python test suite"
 	@echo "  testshell        Runs a shell in the test environment"
-	@echo "  gunicorn         Runs the whole stack using gunicorn on http://localhost:8000/"
-	@echo "  django-shell     Django integrative shell"
```

Comment:
I took out some make rules that can easily be run manually and don't come up often.